README.md

```diff
@@ -42,14 +42,10 @@ Arguments:
 
 ### Get certificate
 
-Pass the certificate type (either `auth` or `sign`) and origin URL as
-JSON-encoded command-line arguments to the `get-certificate`
+Pass the origin URL as JSON-encoded command-line arguments to the `get-certificate`
 command to retrieve the certificate:
 
-    web-eid -c get-certificate '{"type": "auth", "origin": "https://ria.ee"}'
-
-Passing `"type": "auth"` will retrieve the authentication certificate and
-`"type": "sign"` the signing certificate.
+    web-eid -c get-certificate '{"origin": "https://ria.ee"}'
```

Comment:
We discussed this with Tanel. We have to make it clear and explicit that it is the signing certificate, so we decided to rename the command to `get-signing-certificate`.